documentation/reference/test-api/testcontroller/openwindow.html

```diff
@@ -0,0 +1,2256 @@
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/screenshots.html"  >screenshots</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/src.html"  >src</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/startapp.html"  >startApp</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/stop.html"  >stop</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/tsconfigpath.html"  >tsConfigPath</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/useproxy.html"  >useProxy</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/runner/video.html"  >video</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/testcafe/"  >TestCafe</a>
+
+		
+			<ul>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/testcafe/close.html"  >close</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/testcafe/createbrowserconnection.html"  >createBrowserConnection</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/testcafe/createlivemoderunner.html"  >createLiveModeRunner</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/testcafe-api/testcafe/createrunner.html"  >createRunner</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+    <li>
+		    
+		
+
+<span>Plugin API</span>
+
+		
+			<ul>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/plugin-api/browserprovider.html"  >BrowserProvider Interface</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/reference/plugin-api/reporter.html"  >Reporter Interface</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+</ul>
+				</li>
+			</ul>
+		</div>
+	</nav>
+	
+
+	
+	
+	
+
+
+<main>
+	<article class="doc">
+		<div class="post-content">
+
+			<h1>t.openWindow method</h1>
+<p>Opens a new browser window. Returns the matching window descriptor.</p>
+<div class="highlight"><pre><code class="language-js" data-lang="js"><span class="nx">t</span><span class="p">.</span><span class="nx">openWindow</span><span class="p">(</span> <span class="nx">url</span> <span class="p">)</span> <span class="err">→</span> <span class="nb">Object</span>
+</code></pre></div><div class="table-wrapper"><table><thead><tr>
+<th>Parameter</th>
+<th>Type</th>
+<th>Description</th>
+</tr>
+</thead><tbody><tr>
+<td>url</td>
+<td>String</td>
+<td>The URL to open. Can be local or remote, absolute or relative.</td>
```

Comment:
> Can be local or remote.

I guess it is the redurant phrase. Web site can use local resources (ships with web site) and remote (web fonts loaded from CDN, etc.).

The proposed variant: `The URL to open. Can be absolute or relative.`

Reply:
Also, add a note here that the `openWindow` can be used in the chain (returns test controller) or as for getting window descriptor.